.changeset/purple-hairs-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@talend/json-schema-form-core': patch
+---
+
+test
```

Comment:
To be removed I guess

Reply:
In fact because I have updated jsfc build I prefer to really release it and update integration in react-forms